docs/reference/commandline/stack_config.md

```diff
@@ -26,7 +26,7 @@ Options:
       --help            Print usage
 ```
 
-Displays the configuration of a stack.
+Display the configuration of a stack.
```

Comment:
https://github.com/docker/docker/blob/master/docs/reference/commandline/attach.md
> The `docker attach` command allows  ..

https://github.com/docker/docker/blob/master/docs/reference/commandline/build.md
> Builds Docker images ..

https://github.com/docker/docker/blob/master/docs/reference/commandline/deploy.md
> Create and update a stack..

Seems not uniform across the documents.
Personally I like `The docker attach command allows  ..` style
PTAL @mstanleyjones